xonsh/platform.py

```diff
@@ -189,6 +188,27 @@ def scandir(path):
     win_unicode_console = None
 
 
+if ON_WINDOWS:
+    import winreg
+    try:
+        key = winreg.OpenKey(winreg.HKEY_LOCAL_MACHINE, 'SOFTWARE\\GitForWindows')
+        GIT_FOR_WINDOWS_PATH, type = winreg.QueryValueEx(key, "InstallPath")
+    except FileNotFoundError:
+        GIT_FOR_WINDOWS_PATH = None
+
+    # Check that bash is on path otherwise try the default directory
+    # used by Git for windows
+    import subprocess
+    WINDOWS_BASH_COMMAND = 'bash'
+    try:
+        subprocess.check_call([WINDOWS_BASH_COMMAND, '--version'])
```

Comment:
This is showing output for me right when I start up xonsh, with bash installed.  Can we add `stdout=subprocess.PIPE` here?  And maybe `stderr`, too, just to be safe?